src/game.cpp

```diff
@@ -11659,13 +11659,29 @@ void game::pldrive(int x, int y)
         }
     }
 
+    int turn_delta = 15 * x;
+    if (turn_delta != 0) {
```

Comment:
I'm working on getting enough code compliant with the current style to do
that, but it's an uphill battle.